src/nspawn/nspawn-bind-user.c

```diff
@@ -286,6 +286,10 @@ int bind_user_prepare(
                 if (!sd)
                         return log_oom();
 
+                options = strdup("norbind");
```

Comment:
Please describe in the commit message what the intent and purpose of this are…

Reply:
I marked this as draft because this was purely to allow someone to test if this fixes their issue, but they had no time to test and just closed the issue, so I'll close this

Reply:
It would be wonderful when the fix to a known problem would remain to be tracked. 
At least left, so that future contributors can test it.